asciidoc/courses/llm-fundamentals/modules/2-vectors-semantic-search/lessons/3-c-create-vector-index/questions/1-name-the-movie.adoc

```diff
@@ -0,0 +1,25 @@
+[.question.freetext]
+= Name the Movie
+
+What is the **title** of the movie with the closest similarity to the embedding created from the prompt?
+
+input::answer[]
+
+* [x] Meet John Doe
```

Comment:
This question is dependent on the learner not having changed the database during the course. We often see emails from learner unable to get the correct answer because they have "experimented" and modified the data.